docs/releases.md

```diff
@@ -9,28 +9,51 @@
 
 ## Release Philosophy
 
-`go-ipfs` aims to have release every six weeks, two releases per quarter. During these 6 week releases, we go through 4 different stages of Release Candidates (RC) that gives us the opportunity to test the new version against our test environments (unit, interop, integration), QA in our current production environment, IPFS apps (e.g. Desktop and WebUI) and with our _early testers_<sup>[1]</sup> that have IPFS running in Production, by leveraging their own test infrastructure and QA systems.
+`go-ipfs` aims to have release every six weeks, two releases per quarter. During these 6 week releases, we go through 4 different stages that gives us the opportunity to test the new version against our test environments (unit, interop, integration), QA in our current production environment, IPFS apps (e.g. Desktop and WebUI) and with our community and _early testers_<sup>[1]</sup> that have IPFS running in Production.
 
 We might expand the six week release schedule in case of:
 - No new updates to be added
 - In case of a large community event that takes the core team availability away (e.g. IPFS Conf, Dev Meetings, IPFS Camp, etc)
 
 ## Release Flow
 
-`go-ipfs` releases come in 4 stages:
+`go-ipfs` releases come in 4 stages designed to gradually roll out changes and reduce the impact of any regressions that may have been introduced.
 
-- **Release Stage 1 - Internal testing** - Test the Release against our testing infrastructure, including interoperability, integration, test lab, multiple runtimes and the apps we've built (WebUI, Desktop, NPM on IPFS, HTTP Client Libraries). The intent is to make this stage fully automated (and somewhat is already), until then, we manually check a list and ensure that all tests have been run
-- **Release Stage 2 - Invite _early testers_ to try it out** - Reach out to our _early testers_ (i.e. projects that have volunteered to support `go-ipfs` by using their own test infrastructure and tell us the results)
-- **Release Stage 3 - Announce to the broader community** - Communicate to the community that a new Release Candidate is ready and that everyone is welcome to test it with us
-- **Release Stage 4 - Complete the Release** - Finalize the release, start the next release.
+### Alpha
```

Comment:
I've very mixed feelings of overloading terms that are already super overloaded.

IPFS is Alpha software. Having a Alpha release of Alpha software will be confusing, plus moving to a Beta and Final stage of Alpha software will be misunderstood and people will just start talking about IPFS as non Alpha.

Reply:
You're right. This _also_ makes it hard to restart the process because we can't really go from rc1 back to alpha. I'm opening a new PR where stages and versions are independent.